daemon/daemon_unix_test.go

```diff
@@ -319,6 +319,7 @@ func TestVerifyPlatformContainerResources(t *testing.T) {
 			},
 			sysInfo: sysInfo(t, withMemoryLimit),
 			expectedWarnings: []string{
+				"You are using cgroups v2 which does not support OomKillDisable. OomKillDisable discarded.",
 				"Your kernel does not support OomKillDisable. OomKillDisable discarded.",
```

Comment:
I'm a bit confused; shouldn't the warnings contain _either_  "does not support" (v1) or "you are using cgroups v2" (v2), not _both_ ?

Reply:
I think the test should be removed as we cannot test cgroups v1/2 (it's not in SysInfo, perhaps it should be?)

Reply:
We could possibly get it in another way, but I was curious why it passed.

I think there's a bug in the test; I see it's running with `t.Parallel()`, but it doesn't capture the `tc` in the loop; https://github.com/moby/moby/blob/085c6a98d54720e70b28354ccec6da9b1b9e7fcf/daemon/daemon_unix_test.go#L358-L359

Let me open a quick PR to fix that (and see if there's failures after fixing it)